cpp/include/cugraph/sampling_functions.hpp

```diff
@@ -1333,7 +1310,7 @@ heterogeneous_renumber_and_sort_sampled_edgelist(
   raft::handle_t const& handle,
   rmm::device_uvector<vertex_t>&& edgelist_srcs,
   rmm::device_uvector<vertex_t>&& edgelist_dsts,
-  std::optional<rmm::device_uvector<weight_t>>&& edgelist_weights,
+  std::vector<arithmetic_device_uvector_t>&& edgelist_edge_properties,
   std::optional<rmm::device_uvector<edge_id_t>>&& edgelist_edge_ids,
   std::optional<rmm::device_uvector<edge_type_t>>&& edgelist_edge_types,
```

Comment:
Do we still need edgelist_edge_ids & edgelist_edge_types here? Aren't they edgelist_edge_properties?